### ISSUE TITLE
Segfault when building the image.

### ISSUE THREAD

**Issue by jdumas** [willfurnass/abaqus-2017-centos-7-singularity#3]
Hi @willfurnass. Thanks for putting this script out there, it's very useful!

I'm having some trouble running the build script with singularity 3.0. More specifically, I encounter a segfault when installing AbaqusServices `ksh ./2/SIMULIA_AbaqusServices/Linux64/1/StartTUI.sh`:

```
Executing application /usr/DassaultSystemes/SimulationServices/V6R2017x/linux_a64/code/bin/DSYEnvMgt...
*** Error in `/usr/DassaultSystemes/SimulationServices/V6R2017x/linux_a64/code/bin/DSYEnvMgt': free(): invalid pointer: 0x00007f7e1d4667b8 ***
```

I was wondering if you ever had a similar issue, and if so if you have any idea how to solve it? The weird thing is that the installation seems to go through if I use the interactive installation in sandbox mode.

**Comment by willfurnass** [comment]
Yep, I get the following when trying to build non-interactively with Singularity 3.0.2-2 on Arch Linux:

```
Executing application /opt/abaqus/2017/V6R2017x/linux_a64/code/bin/DSYEnvMgt...
*** Error in `/opt/abaqus/2017/V6R2017x/linux_a64/code/bin/DSYEnvMgt': free(): invalid pointer: 0x00007fc02ad4c7b8 ***
```

I hadn't tested this recipe with Singularity 3.x before.  Not sure of the cause of this issue - something related to cgroup-imposed memory limits perhaps?
